pkgs/servers/libreddit/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib
+, stdenv
+, rustPlatform
+, fetchFromGitHub
+, Security
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "libreddit";
+  version = "0.3.0";
+
+  src = fetchFromGitHub {
+    owner = "spikecodes";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "05qs05546b4zz60752xdvvjhihbspifjfs96lvmpw26521n3374h";
+  };
+
+  cargoSha256 = "0gk92bpqp462yrmnlkff311i6rgww4a9pan4xr435q2clajq79lw";
+
+  buildInputs = lib.optional stdenv.isDarwin Security;
```

Comment:
```suggestion
  buildInputs = lib.optional stdenv.isDarwin Security;
  
  passthru = {
    inherit (nixosTests) libreddit;
  };
```

Reply:
Thanks for the feedback.

Reply:
I'm sorry but that suggestion wasn't correct and ofborg didn't run it. Sorry for the inconvenience. It should be:

```
passthru.tests = {
 inherit (nixosTests) libreddit;
};
```

Found it defined as so in other packages (`git grep nixosTests pkgs`).